src/tauon/t_modules/t_main.py

```
@@ -1912,8 +1912,10 @@ def try_reload_playlist_from_file(self, playlist: TauonPlaylist, warnings: bool
 				playlist.playlist_ids[:] = p[:]
 
 			playlist.file_size = path.stat().st_size
-			if stations:
+			try:
 				self.tauon.add_stations(stations, playlist.title)
+			except:
```

Bare except is not desirable.

When would this be throwing an exception that we want to pass?

Catch that specific exception, document it, and add a generic Exception case below it and log it, we don't want to silently ignore all exceptions